tox.ini

```diff
@@ -1,32 +1,34 @@
 [tox]
-envlist = test
-toxworkdir = /var/tmp
+envlist = py27,py3
+toxworkdir = {homedir}/.virtualenvs/docker_utils
+# passenv = DOCKER_HOST DOCKER_CERT_PATH DOCKER_TLS_VERIFY
+
+[testenv:py27]
+basepython = python2.7
+
+[testenv:py3]
+basepython = python3
 
 [testenv]
 # Consolidating all deps here instead of cleanly/separately in test/style/cover so we
 # have a single env (platform) to work with, which makes debugging easier (like which env?).
 # Not as clean but easier to work with for dev, which is better.
 deps =
-    -rrequirements.txt
```

Comment:
how come this is moved to `commands`?

Reply:
See https://github.com/tox-dev/tox/issues/149.
Dependencies were stale when testing and it didn't pick up new `requirements.txt`.